jobs/JEVS_CAM_STATS

```diff
@@ -101,29 +101,28 @@ export MET_PLUS_CONF="${PARMevs}/metplus_config/${STEP}/${COMPONENT}/${VERIF_CAS
 ####################################
 # Define COMIN/COMOUT variables
 ####################################
-export COMIN=${COMIN:-$(compath.py $NET/$evs_ver)}
+export COMIN=${COMIN:-$(compath.py $envir/com/$NET/$evs_ver)}
 export COMINhiresw=${COMINhiresw:-$(compath.py ${envir}/com/hiresw/${hiresw_ver})}
-export COMINhref=${COMINhref:-$(compath.py  href/$href_ver)}
+export COMINhref=${COMINhref:-$(compath.py href/$href_ver)}
 export COMINhrrr=${COMINhrrr:-$(compath.py ${envir}/com/hrrr/${hrrr_ver})}
 export COMINnam=${COMINnam:-$(compath.py ${envir}/com/nam/${nam_ver})}
 export COMINobsproc=${COMINobsproc:-$(compath.py ${envir}/com/obsproc/${obsproc_ver})}
-export COMINmping=${COMINmping:-$(compath.py -o $NET/$evs_ver/prep/$COMPONENT)}
-export COMINprepbufr=${COMINprepbufr:-$(compath.py  obsproc/${obsproc_ver})}
-export COMINsnow=$DCOMROOT
-export COMINspcotlk=${COMINspcotlk:-$(compath.py -o $NET/$evs_ver/prep/$COMPONENT)}
+export COMINprepbufr=${COMINprepbufr:-$(compath.py obsproc/${obsproc_ver})}
```

Comment:
This may be redundant. There is no prepbufr package. prepbufr data is contained inside the obsproc pacakge's com space. This variable should be COMINobsproc instead and you already have that defined in line 109.